GetConfigFiles.py

```diff
@@ -39,8 +39,8 @@ def write_file_from_db(cursor, script):
         if not os.path.isdir(directory):
             print(f"Creating Direcotry {directory}")
             os.makedirs(directory)
-        with open(file, "wb+") as fp:
-            fp.write(rec[1])
+        with open(file, "wb+") as file:
+            file.write(rec[1])
```

Comment:
I'm not sure this change makes sense: with this change, you're shadowing the `file` variable, so what was originally a string is now a an open file object. It will *work*, because Python is a dynamically typed language, but I think the original version was more clear.

Reply:
ok will change.

Reply:
The easiest solution here is to simply delete the commit that contains this change. I've done that and updated this PR.